apps/schools/store/interfaces.ts

```diff
@@ -0,0 +1,11 @@
+export interface ErrorType {
+    data: {
+        error: {
+            code: string,
+            message?: string,
+            'violation_fields': any,
+            violations: [string]
+        }
+    },
+    status: number,
+}
```

Comment:
I don't understand in which directories we keep interfaces and how to manipulate with them
can you explain?

Reply:
I thought thiss interface should be in common/domains